### PR TITLE
Restructure App Development intro

### PR DESCRIPTION
**Pull request by malandrina**

### CODE REVIEW

**Comment by malandrina** [comment]
I'm trying to start the narrative of building an app with geocoding functionality here. It needs a closing sentence that leads nicely into Search Data Locally but nothing's coming to me at the moment.

File: book/application_development/introduction.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
-## Developing a Rails Application with Geocoding
+## Introduction
 
-When building a Rails application, a model will often not be written with any
-type of geocoding. The first pieces of functionality are often to display all
-locations without any mapping or searching capabilities.
+The following chapters outline recommended approaches to building features that
```
Comment (joshuaclayton) [inline book/application_development/introduction.md]: that => which?

File: book/application_development/introduction.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+rely on the presence of geocoded data. As a starting point, let's assume we
+have a few thousand records in our database. Let's also assume that address
```
Comment (joshuaclayton) [inline book/application_development/introduction.md]: can we merge these two sentences? "As a starting point, let's assume we have a few thousand records in our database, each with accurate address information."?

File: book/application_development/introduction.md [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
+information is available for all our records.
 
-Let's look at some code which merely displays location information straight
-from the database:
+Without the ability to query our data by location, all we can do is display
```
Comment (joshuaclayton) [inline book/application_development/introduction.md]: "all we can do is display" => "we can start by displaying"?

File: book/application_development/introduction.md (hunk 2 above)
Comment (malandrina) [inline book/application_development/introduction.md]: Yes! Done.

**Pull request merged**